libs/schema-profunctor/src/Data/Schema.hs

```diff
@@ -563,6 +564,10 @@ instance ToSchema String where schema = genericToSchema
 
 instance ToSchema Bool where schema = genericToSchema
 
+instance (S.ToSchema a, A.ToJSON a, A.FromJSON a) => ToSchema [a] where schema = genericToSchema
+
+instance (S.ToSchema a, A.ToJSON a, A.FromJSON a) => ToSchema (NonEmpty a) where schema = genericToSchema
```

Comment:
I would also like to define this in terms of `ToSchema a` rather than `S.ToSchema a`. It should be possible to use `array`, and then `withParser` to convert it into a non-empty list.